packages/cli/test/Prompt.test.ts

```diff
@@ -0,0 +1,84 @@
+import type * as CliApp from "@effect/cli/CliApp"
+import * as Command from "@effect/cli/Command"
+import * as Prompt from "@effect/cli/Prompt"
+import * as MockConsole from "@effect/cli/test/services/MockConsole"
+import * as MockTerminal from "@effect/cli/test/services/MockTerminal"
+import {} from "@effect/platform"
+import { Array, Effect } from "effect"
+import * as Console from "effect/Console"
+import * as Fiber from "effect/Fiber"
+import * as Layer from "effect/Layer"
+import { describe, expect, it } from "vitest"
+
+const MainLive = Effect.gen(function*(_) {
+  const console = yield* _(MockConsole.make)
+  return Layer.mergeAll(
+    Console.setConsole(console),
+    MockTerminal.layer
+  )
+}).pipe(Layer.unwrapEffect)
+
+const runEffect = <E, A>(
+  self: Effect.Effect<A, E, CliApp.CliApp.Environment>
+): Promise<A> => Effect.provide(self, MainLive).pipe(Effect.runPromise)
+
+describe("Prompt", () => {
+  describe("text", () => {
+    it("should return an empty string when `default` on `Prompt.TextOptions` is not provided", () =>
+      Effect.gen(function*(_) {
+        const prompt = Prompt.text({
+          message: "This should not have a default"
+        })
+
+        const command = Command.prompt(
+          "prompt-command",
+          prompt,
+          (value) =>
+            Console.log(
+              `Prompt value: "${value}"`
+            )
+        )
+
+        const cli = Command.run(command, {
+          name: "Default Value App",
+          version: "0.0.1"
+        })
```

Comment:
Because a `Prompt` is a subtype of `Effect`, you should be able to avoid spinning up a whole CLI application here.

You can just directly `Effect.fork(prompt)`.

Reply:
Perfect. Thanks for the tip. I was able to greatly simplify the tests.